src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -1518,7 +1518,14 @@ void S6Exporter::ExportBanners()
     {
         auto src = GetBanner(i);
         auto dst = &_s6.banners[i];
-        ExportBanner(*dst, *src);
+        if (src != nullptr)
+        {
+            ExportBanner(*dst, *src);
+        }
+        else
+        {
+            ExportBanner(*dst, {});
```

Comment:
This is not correct. If there is no source banner, it should create a banner that RCT2 recognises as a "null banner". Right now, it will just create a banner with everything set to 0, which is not the same.

Reply:
The default constructor for the banner sets it to banner null not 0 so this should be setting it correctly